common/src/main/java/com/microsoft/identity/common/internal/request/SdkType.java

```diff
@@ -22,10 +22,39 @@
 // THE SOFTWARE.
 package com.microsoft.identity.common.internal.request;
 
+import com.microsoft.identity.common.adal.internal.AuthenticationConstants;
+
 /**
  * Enum to indicate if the request came from an app hosting Adal or Msal Sdk
  */
 public enum SdkType {
     ADAL,
-    MSAL
-}
\ No newline at end of file
+    MSAL,
+    MSAL_CPP,
+    UNKNOWN;
+
+    /**
+     * Method for mapping the SdkType to appropriate String
+     * for the purpose of sending it to the telemetry.
+     */
```

Comment:
return not documented

Reply:
c361c635cdbd853343a0ed18cc0b426d9c508037